x-pack/packages/kbn-elastic-assistant/impl/assistant/helpers.ts

```diff
@@ -122,10 +122,4 @@ export const getOptionalRequestParams = ({
   };
 };
 
-export const hasParsableResponse = ({
-  isEnabledRAGAlerts,
-  isEnabledKnowledgeBase,
-}: {
-  isEnabledRAGAlerts: boolean;
-  isEnabledKnowledgeBase: boolean;
-}): boolean => isEnabledKnowledgeBase || isEnabledRAGAlerts;
+export const sleep = (ms: number) => new Promise((resolve) => setTimeout(resolve, ms));
```

Comment:
Side note: There's some utils for retries floating around if we end up needing more complex retry logic. Here's what I used in the KB setup logic:

https://github.com/elastic/kibana/blob/b6fcfac9c104bbaacb7e9f3250b55c7f5e13eeae/x-pack/plugins/elastic_assistant/server/ai_assistant_data_clients/knowledge_base/index.ts#L181-L187